lib/Twig/TemplateWrapper.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2016 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Exposes a template to userland.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class Twig_TemplateWrapper
```

Comment:
Should we really call it a wrapper ? The fact that it is a wrapper around a Twig_Template object is also an implementation detail. I'm wondering whether we can find a better name here (I have no good idea though)